examples/LAPACK/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-LAPACK)
+
+hunter_add_package(LAPACK)
+
+string(COMPARE EQUAL "${LAPACK_LICENSES}" "" is_empty)
+if(is_empty)
+  message(FATAL_ERROR "Licenses not found")
+endif()
+
+message("LAPACK licenses:")
+foreach(x ${LAPACK_LICENSES})
+  message("* ${x}")
+  if(NOT EXISTS "${LAPACK_LICENSES}")
+    message(FATAL_ERROR "File not found")
+  endif()
+endforeach()
+
+find_package(LAPACK CONFIG REQUIRED)
+
+# Test double library creation
+find_package(LAPACK CONFIG REQUIRED)
+
+
+find_package(cblas CONFIG REQUIRED)
```

Comment:
Since this pull request is about adding `lapack` I expect at least one simple example with `hunter_add_package(lapack)` + `find_package(lapack ...)` + `#include <lapack.h>` without any other instructions. If there are other important variants of using this package they should be added separately, e.g. `examples/lapack-cblas`. In those examples we can use custom local `config.cmake` with custom `CMAKE_ARGS` for enabling/disabling components.